common/cf-guest/src/client.rs

```diff
@@ -30,6 +30,22 @@ pub struct ClientState<PK> {
 }
 
 impl<PK: guestchain::PubKey> ClientState<PK> {
+    pub fn new(
+		genesis_hash: CryptoHash,
+		latest_height: guestchain::BlockHeight,
+		trusting_period_ns: u64,
+		epoch_commitment: CryptoHash,
+		is_frozen: bool,
+	) -> Self {
+		Self {
+			genesis_hash,
+			latest_height,
+			trusting_period_ns,
+			epoch_commitment,
+			is_frozen,
+			_ph: core::marker::PhantomData::<PK>,
```

Comment:
```suggestion
			_ph: core::marker::PhantomData,
```
should just work, no?

Reply:
well yes